gpu-operator/life-cycle-policy.rst

```diff
@@ -89,36 +89,37 @@ Refer to :ref:`Upgrading the NVIDIA GPU Operator` for more information.
      - ${version}
 
    * - NVIDIA GPU Driver
-     - | `570.133.20 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-133-20/index.html>`_ (recommended), 
-       | `570.124.06 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-124-06/index.html>`_ (default)
-       | `570.86.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-86-15/index.html>`_ 
+     - | `570.148.08 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-148-08/index.html>`_ (default, recommended)
+       | `570.133.20 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-133-20/index.html>`_
+       | `570.124.06 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-124-06/index.html>`_
+       | `570.86.15 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-570-86-15/index.html>`_
        | `550.163.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-550-163-01/index.html>`_
        | `535.247.01 <https://docs.nvidia.com/datacenter/tesla/tesla-release-notes-535-247-01/index.html>`_
 
    * - NVIDIA Driver Manager for Kubernetes
      - `v0.8.0 <https://ngc.nvidia.com/catalog/containers/nvidia:cloud-native:k8s-driver-manager>`__
 
    * - NVIDIA Container Toolkit
-     - `1.17.5 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
+     - `1.17.8 <https://github.com/NVIDIA/nvidia-container-toolkit/releases>`__
 
    * - NVIDIA Kubernetes Device Plugin
-     - `0.17.1 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
+     - `0.17.2 <https://github.com/NVIDIA/k8s-device-plugin/releases>`__
 
    * - DCGM Exporter
-     - `4.1.1-4.0.4 <https://github.com/NVIDIA/dcgm-exporter/releases>`__
+     - `4.2.3-4.1.1 <https://github.com/NVIDIA/dcgm-exporter/releases>`__
```

Comment:
One more update to dcgm-exporter was added to 25.3.1
```suggestion
     - `4.2.3-4.1.3 <https://github.com/NVIDIA/dcgm-exporter/releases>`__
```

Reply:
updated